webapp/channels/src/components/initial_loading_screen/initial_loading_screen_class.ts

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const ANIMATION_CLASS_FOR_MATTERMOST_LOGO_HIDE = 'LoadingAnimation__compass-shrink';
+const ANIMATION_CLASS_FOR_COMPLETE_LOADER_HIDE = 'LoadingAnimation__shrink';
+
+const LOADING_CLASS_FOR_SCREEN = 'LoadingScreen LoadingScreen--darkMode';
+const LOADING_COMPLETE_CLASS_FOR_SCREEN = 'LoadingScreen LoadingScreen--darkMode LoadingScreen--loaded';
+
+const STATIC_CLASS_FOR_ANIMATION = 'LoadingAnimation LoadingAnimation--darkMode';
+const LOADING_CLASS_FOR_ANIMATION = STATIC_CLASS_FOR_ANIMATION + ' LoadingAnimation--spinning LoadingAnimation--loading';
+const LOADING_COMPLETE_CLASS_FOR_ANIMATION = STATIC_CLASS_FOR_ANIMATION + ' LoadingAnimation--spinning LoadingAnimation--loaded';
+
+export class InitialLoadingScreenClass {
+    private isLoading = true;
+
+    private loadingScreenElement?: HTMLElement | null;
+    private loadingAnimationElement?: HTMLElement | null;
+
+    private initialLoadingScreenCSS?: HTMLLinkElement | null;
```

Comment:
Nit: Since these are always defined in the constructor, do these need to be optional or can we remove the `?` from them?

Reply:
Yes i removed the optional